src/main/java/com/minecolonies/coremod/configuration/ConfigurationHandler.java

```diff
@@ -143,7 +136,33 @@ private static synchronized void loadConfiguration()
             }
         }
     }
-    
+
+    /**
+     * load configuration related to Path finding.
+     */
+    private static synchronized void loadPathFindingConfigurations()
+    {
+        pathfindingDebugDraw = config.get(CATEGORY_PATHFINDING, "debugDraw", pathfindingDebugDraw,
+                "Render pathfinding results for debugging purposes (SSP only)").getBoolean();
+        pathfindingDebugVerbosity = config.get(CATEGORY_PATHFINDING, "debugVerbosity", pathfindingDebugVerbosity,
+                "Debug output verbosity of pathfinding (0=none, 1=results, 2=live work)").getInt();
+        pathfindingMaxThreadCount = config.get(CATEGORY_PATHFINDING, "maxThreads", pathfindingMaxThreadCount,
+                "Maximum number of threads to use for pathfinding.").getInt();
+    }
+
+    /**
+     * load configuration related to the citizen's names.
+     */
+    private static synchronized void loadNamesConfigurations()
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused private "loadNamesConfigurations" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUnusedPrivateMethod)